src/AbstractEvent.php

```diff
@@ -221,5 +224,15 @@ public function removeParticipation(EventParticipation $participation)
 
         return $this;
     }
+
+    public function setRaw(array $raw)
+    {
+        $this->raw = $raw;
+    }
```

Comment:
you should remove this method, I don't see the point of modifying the `raw` property except inside its own hydrators